### PR TITLE
Optimization for obsidian users

### PR DESCRIPTION
**Pull request by rab-bit**
Hi @nixee,

Firstly, my big thanks for the script. 
I've made some changes while migrating to Obsidian that could be interesting for you. Check out the readme to more information.

Regards

rab-bit

### CODE REVIEW

**Comment by winstonleedev** [comment]
Hey, I'm not the author, but thanks for the update. Works really well!

Can you change line 286 to say `--markdown-headings=atx` instead of `--atx-headers`? It seems pandoc recently changed its parameters. It's too small for a separate pull request :)

**Comment by winstonleedev** [comment]
Also, try submitting your pull request to the original repository https://github.com/SjoerdV/ConvertOneNote2MarkDown It seems @nixsee is not active on github since last October

**Comment by rab-bit** [comment]
Hi @thanhphu, thanks for the hints. I've updated the pandoc's parameter and sendet a pull request to the original repository.

**Comment by nixsee** [comment]
Hey @rab-bit, sorry for being MIA. Glad to see that the script has been helpful to others and thanks for all of these additions! I'm a complete hack, so without really thinking about this for a while, its not clear to me what this adds/subtracts from the original. If there's any pure addition of functionality, I'm all for it. But it appears to replace certain things, which I'm not interested in merging without understanding them first. However, if your replacements could be added as a menu option to select from - e.g. 1 for original 2 for MOC structure, that would be great. 

Could you summarize the additions briefly? Thanks!

p.s. I'm pretty keen about the MOC/LYT framework as well, so look forward to understanding this better and perhaps implementing it myself.

**Comment by nixsee** [comment]
@rab-bit, please address my questions above in order for me to consider merging this pull request.

**Comment by rab-bit** [comment]
Hi @nixsee. Sorry, it has been a long time since I made the changes, and I am not able to detailed explain all the differences anymore. The new Features are described under https://github.com/rab-bit/ConvertOneNote2MarkDown4Obsidian#added-features. The modifications can actually only be followed by the comments on each commit.  I've also tried to separate each new feature on new methods in code, for the case other users want to adapt it to their solutions. Feel free to use any of the new contents (don't necessarily merging them to the original). If you have any questions about any specific modifications, I would be happy to help.

**Comment by nixsee** [comment]
Thanks, I think I'll reject the pull request then - there is simply far too much unexplained changes to the code and if you're not able to make sense of it, then I'm certainly not going to try. I hope people who need such features will find your fork
